src/kindlemint/automation/kdp_automation_engine.py

```
@@ -0,0 +1,605 @@
+            metadata = self.generate_book_metadata(niche)
+
+            # 2. Generate book content using existing system
+            from scripts.generate_book import generate_book
+
+            pdf_path = await asyncio.to_thread(generate_book)
+
+            # 3. Setup browser automation
+            self.browser_bot.setup_driver()
+
+            # 4. Login to KDP (credentials from environment)
+            kdp_email = os.getenv("KDP_EMAIL")
```

Ruff (0.11.9)

Undefined name `os`

(F821)